aether-kernel/aether/kernel/api/models.py

```diff
@@ -304,7 +304,7 @@ class Attachment(ExportModelOperationsMixin('kernel_attachment'), ProjectChildAb
 
     # http://www.linfo.org/file_name.html
     # Modern Unix-like systems support long file names, usually up to 255 bytes in length.
```

Comment:
So we should probably take out this comment too, shouldn't we?

Reply:
Right!